test/test-applications/integrations/dependency-libs/TestApplication.Shared/TestApplication.Shared.csproj

```diff
@@ -6,8 +6,4 @@
     <StartupObject />
   </PropertyGroup>
 
```

Comment:
Which projects still need this `TestApplication.Shared`?

Reply:
Same question. The methods in this project are still getting called in other projects, e.g. `ConsoleHelper.WriteSplashScreen` is called in `TestApplication.CustomSdk`. I'm surprised that the CI passes despite of this.

Reply:
There is not traditional reference to this project, but still all `TestApplication`s are referencing it.
See  6a1fc5b04366ca789e5e9452315fc7de5e7337d8 for details.

The reason - I was not able to execute restore/build process for multi-lib scenarios with this dependency.
Shared lib was build a lot of times.
It can be improved in the future if needed.

Reply:
Could change the csproj project type.

Did not find better docs reference for this feature. Basically a project type for shared code and not compiled.
https://learn.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/shared-projects?tabs=windows

Reply:
Shared projects based on old style files. The current approach is used by .NET SDK/Contrib repository

```xml
<?xml version="1.0" encoding="utf-8"?>
<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <PropertyGroup Label="Globals">
    <ProjectGuid>30648dda-7487-4754-b6aa-a9a0c87a57f7</ProjectGuid>
    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
  </PropertyGroup>
  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.Default.props" />
  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.props" />
  <PropertyGroup />
  <Import Project="SharedProject1.projitems" Label="Shared" />
  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.CSharp.targets" />
</Project>
```